docs/troubleshooting.adoc

```diff
@@ -0,0 +1,58 @@
+# Troubleshooting walkthroughs
+
+This document contains a set of common issues and errors that are encountered
+when adding a walkthrough to the Integreatly web app.
+
+## Walkthrough is not showing up in the dashboard
+
+This can be due to a number of reasons:
+
+- `WALKTHROUGH_LOCATIONS` does not contain a repo or path with the walkthrough.
+  * Ensure `WALKTHROUGH_LOCATIONS` contains a path or a repo with the walkthrough inside of it.
+  * If in a repo, ensure the walkthrough is in the `walkthroughs` directory of the
+  repo. 
+- The repo containing the walkthrough in `WALKTHROUGH_LOCATIONS` was updated after the server has started.
+  * If the repo was updated after the server has started, invoke the `POST /sync-walkthroughs` endpoint.
+
+## ServiceInstances are not being created
+
+This issue can occur for a number of reasons:
+
+- The walkthrough wasn't started the web app.
+  * ServiceInstances are provisioned when a task in a walkthrough is loaded. Ensure you're not on the walkthrough overview.
+- The ServiceInstance spec is invalid.
+  * The `serviceInstances` object in `walkthrough.json` should be an array of
+    `ServiceInstance` resources. For example:
+
+## Images and videos are not rendering
+
+This issue can occur for a number of reasons:
+
+- The media isn't contained in the walkthrough directory.
+  * If the media isn't inside the directory of the walkthrough it may not be
+  included when the walkthrough is hosted in the web app. It is recommended to
+  bundle any media with the walkthrough.
+- The Asciidoc `image` and `video` tags aren't being used.
+  * If images or videos are referenced using, for example, `<img/>` tags then
+  the path to the media may not be built properly when hosted in the web app. It
+  is recommended to use the Asciidoc `image` and `video` tags.
```

Comment:
is there somewhere we can link out to that explains these tags,
or maybe give an example inline here?